lib/fluent/plugin/in_tail.rb

```diff
@@ -117,6 +114,30 @@ def configure_tag
       end
     end
 
+    def configure_encoding
+      if @encoding
+        @encoding = parse_encoding_param(@encoding)
+        if @from_encoding
+          @from_encoding = parse_encoding_param(@from_encoding)
+        end
+      else
+        if @from_encoding
+          $log.warn "'from_encoding' parameter must be specified in conjunction with 'encoding' paramter."
+          $log.warn "these parameter set to nil."
```

Comment:
It looks better to show `"from_encoding" is ignored` if it is specified without `encoding`.
"set to nil" explains nothing for users (users doesn't know anything about implementation).